src/rulesets/Base/fastmath_able.jl

```diff
@@ -88,3 +112,7 @@ let
     eval(fastable_ast)  # Get original definitions
     # we do this second so it overwrites anything we included by mistake in the fastable
 end
+
+_sign_jvp(Ω, absz, Δ) = Ω * ((imag(Δ) * real(Ω) - real(Δ) * imag(Ω)) / absz)im
```

Comment:
A brief comment here explaining why this function is here / what it does would be helpful.